app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
+    total_rating = 0
```

Comment:
you can do it in a one line, but it's up to you

Reply:
Fixed